packages/editor/src/components/projects/ProjectsPage.tsx

```diff
@@ -212,7 +213,11 @@ const ProjectsPage = ({ studioPath }: { studioPath: string }) => {
     }
   })
 
-  const installedProjects = projectFindQuery.data.filter(() => projectCategoryFilter.installed.value)
+  const isOwner =
+    projectPermissionsFindQuery.data?.find((permission) => permission.userId === selfUser.id.value)?.type === 'owner'
```

Comment:
I'm assuming we also want to show projects that you have `editor` permissions on. I am unsure what to do about `reviewer` permissions